binderhub/builder.py

```diff
@@ -500,7 +538,7 @@ def _check_result(future):
                     elif progress.payload == ProgressEvent.BuildStatus.BUILT:
                         event = {
                             "phase": phase,
-                            "message": "Built image, launching...\n",
+                            "message": "Done! Image built.\n",
```

Comment:
I'm a bit worried that we change to much by changing this frequently seen log messages. It feels like a part of the binderhub experience at this point. I'm so used to `Built image, launching...` and would be a bit uncertain on what was going on if I just saw `Done! Image built.` instead, especially without any further messages about waiting for a launch.

I consider the options of:
- making the messages exactly like they were before this feature was around
- breaking apart the message `Built image, launching...` into two parts, one about the image being built, and one about launching or being done.